synapse/res/templates/_base.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}{% endblock %}</title>
+    <link rel="stylesheet" href="./style.css">
```

Comment:
I think this means "replace the last URL path segment with style.css", like a unix file system?

Where does this stylesheet get served? When Synapse dynamically renders the templates below, do they all get served at a URL such that this stylesheet can be found?

Additionally: I think one usually adds `type="text/css"` to a stylesheet `<link>` (though I expect this is probably not needed for browsers to make sense of it.)

Reply:
Ah! Yes this is incorrect, I think. Good catch!